packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -67,12 +67,12 @@ function MaybeIframe( { children, contentRef, shouldIframe, styles, style } ) {
 
 	return (
 		<Iframe
-			head={ <EditorStyles styles={ styles } /> }
```

Comment:
There are a few more places where `<EditorStyles>` appears in the `head` of an `Iframe`, but I think they can be fixed in other follow-up PRs.

Reply:
Just double-checking — is the main reason to include the svg filters within `EditorStyles` so that folks don't have to know to add the filters in separately when constructing an iframe?

Reply:
I have the impression that we wanted the styles in the `head` for some reasons.. Maybe @ellatrix has some thoughts..

Reply:
> I think they can be fixed in other follow-up PRs.

We're moving them into this one. https://github.com/WordPress/gutenberg/pull/49239#issuecomment-1480763475

> I have the impression that we wanted the styles in the head for some reasons.. Maybe @ellatrix has some thoughts..

See https://github.com/WordPress/gutenberg/pull/49239#discussion_r1145752251

Reply:
> Conceptually, SVG filters are styles, they just aren't CSS.

Thanks, that helps clarify it for me 👍